state-chain/pallets/cf-ingress-egress/src/migrations.rs

```diff
@@ -1,12 +1,4 @@
 use crate::Pallet;
-use cf_runtime_upgrade_utilities::{PlaceholderMigration, VersionedMigration};
-mod add_refund_params;
-pub mod remove_prewitnessed_deposits;
-pub mod withheld_transaction_fees;
+use cf_runtime_upgrade_utilities::PlaceholderMigration;
 
-pub type PalletMigration<T, I> = (
-	VersionedMigration<Pallet<T, I>, remove_prewitnessed_deposits::Migration<T, I>, 8, 9>,
-	VersionedMigration<Pallet<T, I>, add_refund_params::Migration<T, I>, 9, 10>,
-	VersionedMigration<Pallet<T, I>, withheld_transaction_fees::Migration<T, I>, 10, 11>,
-	PlaceholderMigration<Pallet<T, I>, 11>,
-);
+pub type PalletMigration<T, I> = PlaceholderMigration<Pallet<T, I>, 11>;
```

Comment:
For ingress-egress there's a 11 -> 12 only in the release branch. 
https://github.com/chainflip-io/chainflip-backend/blob/d0f687b3891f37ca138ea1e76b2e5eb91e19414a/state-chain/pallets/cf-ingress-egress/src/migrations.rs#L12

Reply:
I added an 11-12 placeholder migration on main - but we can probably delete any migration before version 12 now - would rebase here, since I also bumped the storage version to 12 on main, since that's what it is on release/1.5